src/Services/Services.Common/JsonHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Text.Json;
+
+namespace Services.Common;
+
+public static class JsonHelper
```

Comment:
```suggestion
public static class JsonDefaults
```

Reply:
I was thinking `JsonHelper` would leave it open to more things than "defaults" in the future. But we can rename it if/when that happens.